src/pages/patientView/PatientViewPage.tsx

```diff
@@ -41,6 +44,13 @@ export interface IPatientViewPageProps {
 
 export type MrnaRankData = { [sampleId:string]: { [entrezGeneId:string]: {percentile:number, zScore:number}}};
 export type MutSigData = { [entrezGeneId:string]:{ qValue:number } }
+export type VariantCountData = {
```

Comment:
done, I also moved the definition into CohortColumnFormatter.tsx like @onursumer and I talked about